unfoldingWord/translationCore
# Components core cleanup #1774

#### This pull request addresses:

Cleanup of core components.


#### How to test this pull request:

Make sure all components still render.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1794)
